docs/features/publishing-events.md

```diff
@@ -29,9 +33,42 @@ var @event = new NewCarRegistered(eventId, eventSubject, licensePlate);
 await eventGridPublisher.Publish(@event);
 ```
 
-Alternatively you can publish a list of events by using 
+Alternatively you can publish a list of events by using
+
 ```csharp
 await eventGridPublisher.PublishMany(events);
 ```
 
+### Resilient Publishing
```

Comment:
Maybe makes more sense to rename to `Configuring Resiliency` but not required

Reply:
Hmm, if you don't mind; I would leave it like this. Seems a more appealing title to me; but that could just be me 😅 .

Reply:
Fine for me!